README.md

```diff
@@ -24,13 +24,15 @@ Work items, tickets, etc. are referenced as "issues" in this action.
-| `jql-query`                        | conditionally* | The JQL query to use to find issues that will be transitioned. A max of 20 issues can be transitioned.                                                                                                                                       |
+| `jql-query`                        | conditionally* | The [JQL query](https://support.atlassian.com/jira-software-cloud/docs/jql-operators/) to use to find issues that will be transitioned. A max of 20 issues can be transitioned.                                                              |
 | `issues`                           | conditionally* | Comma delimited list of issues to transition. Use `im-open/get-workitems-action` to identify list of issues for a PR or deployment.                                                                                                          |
 | `transition-name`                  | true           | The name of the transition to perform. _Examples might include Open, In Progress, Deployed, etc._                                                                                                                                            |
 | `update-fields`                    | false          | A [map](#updating-fields) of issue screen fields to overwrite, specifying the sub-field to update and its static value(s) for each field. When multiple sub-fields or other operations are required, use 'process-operations' input instead. |
 | `process-operations`               | false          | A [map](#updating-fields) containing the field name and a list of operations to perform. _The fields included in here cannot be included in 'update-fields' input._                                                                          |
 | `comment`                          | false          | Add a comment to the issue after the transition.                                                                                                                                                                                             |
 | `missing-transition-as-successful` | false          | Mark as a successful if issue is missing the transition. _`true` by default._                                                                                                                                                                |
+| `create-notice`                    | false          | Add notification to runner with details about successful transitioned issues. _`true` by default._                                                                                                                                           |
+| `create-warnings`                  | false          | Add warning notifications to runner. _`true` by default._                                                                                                                                                                                    |
```

Comment:
I don't necessarily mean in this pull request, but enough of these parameters have default values now that it would make sense to add a column to this table for defaults like we do in a lot of other im-open actions.

Reply:
Ya good idea.  I can do that in an upcoming PR.